drivers/media/platform/msm/camera_v2/sensor/msm_sensor_driver.c

```diff
@@ -35,6 +35,62 @@ static int32_t msm_sensor_driver_platform_probe(struct platform_device *pdev);
 /* Static declaration */
 static struct msm_sensor_ctrl_t *g_sctrl[MAX_CAMERAS];
 
+// Power up/down settings obtained from LGH850v10E logs
+// Should be moved into a driver file like imx234.c
+// imx234.c settings do not seem to be parsed 
+// even when "obj-$(CONFIG_IMX234) += imx234.o" is enabled in the Makefile
```

Comment:
The sensor specific drivers (ex. imx234) are no longer used. msm_sensor probe's all the cameras now.